tests/suite/cli_rustup.rs

```diff
@@ -661,7 +661,6 @@ nightly-{0} (active, default)
 active toolchain
 ----------------
 name: nightly-{0}
-compiler: 1.3.0 (hash-nightly-2)
```

Comment:
I would be inclined to show the version even without --verbose.

Reply:
@djc The original rationale to put the rustc version in `--verbose` is that it requires a subprocess call. See also `rustup --version --verbose`.

Reply:
Fair enough!

Reply:
Fair enough!